lib/modules/platform/scm-manager/mapper.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { mapPrFromScmToRenovate } from './mapper';
+import type { PullRequest as SCMPullRequest } from './types';
+
+describe('modules/platform/scm-manager/mapper', () => {
+  it('should correctly map the scm type of a PR to the Renovate PR type', () => {
```

Comment:
```suggestion
  it('should correctly map the scm-manager type of a PR to the Renovate PR type', () => {
```

Did you mean this, perhaps?